meson.build

```diff
@@ -0,0 +1,85 @@
+project('jansson', 'c', version : '2.11', license : 'MIT')
+
+cc = meson.get_compiler('c')
+cdata = configuration_data()
+check_headers = [
+  ['HAVE_DLFCN_H', 'dlfcn.h'],
+  ['HAVE_ENDIAN_H', 'endian.h'],
+  ['HAVE_FCNTL_H', 'fcntl.h'],
+  ['HAVE_INTTYPES_H', 'inttypes.h'],
+  ['HAVE_LOCALE_H', 'locale.h'],
+  ['HAVE_MEMORY_H', 'memory.h'],
+  ['HAVE_SCHED_H', 'sched.h'],
+  ['HAVE_STDINT_H', 'stdint.h'],
+  ['HAVE_STDLIB_H', 'stdlib.h'],
+  ['HAVE_STRINGS_H', 'strings.h'],
+  ['HAVE_STRING_H', 'string.h'],
+  ['HAVE_SYS_PARAM_H', 'sys/param.h'],
+  ['HAVE_SYS_STAT_H', 'sys/stat.h'],
+  ['HAVE_SYS_TIME_H', 'sys/time.h'],
+  ['HAVE_SYS_TYPES_H', 'sys/types.h'],
+  ['HAVE_UNISTD_H', 'unistd.h'],
+]
+
+foreach h : check_headers
+  if cc.has_header(h.get(1))
+    cdata.set(h.get(0), 1)
+  endif
+endforeach
+
+check_functions = [
+# check token ('HAVE_ATOMIC_BUILTINS',)
```

Comment:
Is it TODO?

Reply:
Kinda, yes. All I did was to use the `ac_converter.py`. It seems these tokens are not yet implemented or lesser known and missing from the converter tool. I have to say I have no idea how to implement these specific checks.. :(